scripts/policy/protocols/conn/known-services.zeek

```diff
@@ -210,10 +264,15 @@ event connection_state_remove(c: connection) &priority=-5
 	if ( c$known_services_done )
 		return;
 
-	if ( c$resp$state != TCP_ESTABLISHED )
+	# log tcp connection if established or udp if active
+	if ( c$resp$state != TCP_ESTABLISHED && c$resp$state != UDP_ACTIVE)
+		return;
```

Comment:
Is this change directly related to multi-protocol service handling or something else?  May be better to separate into another PR for clarity if it's related to something else.

Reply:
ok